### DISCUSSION TITLE
Provide shorthand for subcommands

### DISCUSSION THREAD

**Discussion by cnsino** [pallets/click#2513] [Q&A]
How can I provide shorthand for subcommands，such `a.py`
```
@click.command()
def run():
    """Run a new package"""
    click.echo(f'Running of the local package')
```
Add a shorthand `r` to this code to make` python a.py run` consistent with `python a.py r`,The help information is as follows：
```
Commands:
 run,r      Run a new package
```

**Comment by SpaceshipOperations** [comment]
Unfortunately it's not supported in click, and was rejected despite many requests to implement it.

However, there is a [third-party plugin](https://github.com/click-contrib/click-aliases) that adds this ability.

The same account that hosts this plugin also contains [many other useful plugins](https://github.com/click-contrib) that you might want to check out. For example, [click-option-group](https://github.com/click-contrib/click-option-group) (another important and nice feature that is sadly missing from vanilla click).
